### DISCUSSION TITLE
Enable Defecation in OpenRCT2

### DISCUSSION THREAD

**Discussion by raz-a** [OpenRCT2/OpenRCT2#19319] [Feature requests]
I know this sounds like a strange feature request, but the idea is to add more obvious, "real world" consequences to neglecting guests at the park. 

The "Toilet" stat can affect happiness and overall park rating, but in the end, if a guest cannot reach a bathroom, there is no additional consequence.  

On the other hand, nausea has a very obvious effect if not treated. There is a chance that a nauseous guest will vomit, creating path litter and now affecting additional guests who stand on it.  

I suggest we make the Toilet stat have a similar effect to put value into the stat in the first place. 

**I already designed the change here in this currently closed PR: https://github.com/OpenRCT2/OpenRCT2/pull/19312** 

_Note that this should be a very rare occurrence as most people realistically would try to hold it in or leave the park before resorting to public defecation._

Additional "real world" consequences that add value would be something along the lines of, line skippers and fighting guests (giving the security guard a new role to stop these guests), falling asleep guests when they are lo on energy, and guests passing out from dehydration based on thirst. This feature request is specifically for the toilet stat, but just wanted to explain the motivatino with additional examples.

**Comment by ocalhoun6** [comment]
Seems like something that should *realistically* only happen if you have no toilets available *and* trap guests in your park ... which is something that shouldn't be happening in normal gameplay anyway. (And having guests trapped in your park, unable to ever leave, isn't a very realistic scenario anyway, so...?)

>line skippers and fighting guests (giving the security guard a new role to stop these guests)

Doing the coding for line skippers seems like it would be a huge headache. 

If you want more work for security guards, I think a more reasonable way to do it would be to give guests a chance of committing vandalism anytime their energy is high and their happiness is low. (Instead of *only* when they see disgusting paths, as it currently works.) Currently, as long as you keep your paths clean, you don't have to worry about security guards at all because guests only do vandalism when they see dirty paths. If *any* unhappy guest with enough energy could commit vandalism, then it could happen occasionally regardless, which means you'd still want security guards around to prevent it. ---- However, that will also make vandalism more common, so I think to balance that out, you should also have handymen repair vandalized items, perhaps taking a substantial amount of time with each one, but allowing them to do it completely autonomously. That way, the vandalism wouldn't get *completely* out of hand in a large park.

**Comment by 733737** [comment]
I think that the idea makes no sense for a game that is played by people of all ages
